android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -81,7 +81,7 @@
         String ON_CAMERA_SWITCHED          = "onCameraSwitched";
         String ON_VIDEO_CHANGED            = "onVideoChanged";
         String ON_AUDIO_CHANGED            = "onAudioChanged";
-        String ON_CONNECTED                = "onConnected";
+        String ON_CONNECTED                = "onRoomDidConnect";
```

Comment:
The javascript was wrapping it as onConnected.  You can remove that block if you change it here.